doc/TermIt-UsersAndTargets.markdown

```diff
@@ -0,0 +1,100 @@
+# Uživatelé TermIta a jejich očekávání
+
+Nástroj pro správu pojmosloví.
+
+
+Use casy ze skutečného světa jsou:
```

Comment:
Use-cases k přidání
- Covid portal (PK)
- CSAT, UCL, MSMT (MB)

Reply:
- kontexty PPDF (spíš celá VL, ale Termit jako takový taky - řešili jsme např. situaci, kdy si úřad vymyslí XMLko, ale nikdo pořádně netuší, co znamenají XML elementy v tom XMLku, resp. co hůř každý si myslí, že ví, ale každý to nakonec myslí jinak)
- projektové žádosti TAČR (řešil jsem na začátku KODI modelování s TAČR a tam se jim to líbilo jako nástroj pro ujednocení terminologie kolem projektů a projektových žádostí /třeba žadatel, příjemce, řešitel, ...., projektový výstup, projektový výsledek/ - potřebovali vazbu na legislativu i na svojí vnitřní legislativu; dále zmiňovali usecase pro interní využití - pro nové zaměstnance, kteří si slovník projdou a pak se snáz mohou bavit s ostatními, je tam pak rychlejší náběh nových zaměstnanců)